.github/workflows/expose_link.yml

```
@@ -1,4 +1,7 @@
 name: Expose result artifact
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
```

Looking at the [doc] (https://docs.github.com/en/actions/using-jobs/using-concurrency), it seems that we can simply put:

```suggestion
  group: ${{ github.workflow }}-${{ github.ref }}
```